apps/site/assets/ts/schedule/components/Map.tsx

```diff
@@ -33,23 +33,23 @@ export const getBounds = (markers: Marker[]): LatLngBounds => {
 export const iconOpts = (
   icon: string | null
 ): {
-  iconSize?: [number, number];
-  iconAnchor?: [number, number];
+  icon_size?: [number, number];
```

Comment:
what is the reasoning behind this change?

Reply:
So we could set sizes and anchors serverside.

The more generic `<Map />` (the one rendered by this map) now expects the icon_size/icon_anchor to possibly come in the serverside (before we didn't need to), so I made these underscores for consistency (and so that our shared leaflet `<Map />` didn't have to know about it when building icons).

Reply:
I don't love the whole underscore -> camelcase issue in the first place but probably out of scope for this PR.